CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.0-nullsafety.0
+
+* Migrate to null safety.
```

Comment:
We should mention the behavior change here around not returning null work responses, and instead failed responses with broken pipe exit code.

Reply:
Done, feel free to suggest a change